mountpoint-s3-fs/examples/prefetch_benchmark.rs

```diff
@@ -162,8 +162,9 @@ fn main() {
                         let mut offset = 0;
                         while offset < size {
                             let bytes = request.read(offset, args.read_size).await.unwrap();
-                            offset += bytes.len() as u64;
-                            received_bytes.fetch_add(bytes.len() as u64, Ordering::SeqCst);
+                            let length = bytes.len() as u64;
+                            offset += length;
+                            received_bytes.fetch_add(length, Ordering::SeqCst);
```

Comment:
I have some logging changes I'm using for benchmarking, this makes it a bit nicer since I need to reuse the length again.